pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
 
 ## Introduction
 
-In this page the different network components are described in terms of electrotechnical representation.
-Each component is identified through a unique ID, and optionally by a name that is easier to interpret for a human.
-Note that the equipments in the IIDM model may be flagged as fictitious, in order to fine tune the network modelling.
+PowSyBl uses an internal grid model initially developed under the iTesla project, a research project funded by the European Union 7th Framework programme (FP7). The grid model is known as iIDM (iTesla Internal Data Model). One of the iTesla outputs was a toolbox designed to support the decision-making process of power system operation from two-days ahead to real time. The iIDM grid model was at the center of the toolbox.
 
-<span style="color:red"> TODO: when we have aliases, add a description here too.</span>
+To build an electrical network model using iIDM first the substations must be defined. The equipment in a substation (loads, generators, shunts, Static Var Compensators, DC converters ...) are grouped in voltage levels. Transformers present in a substation connect its different voltage levels. Transmission lines (AC and DC) connect the substations. A connection point for an equipment is a _terminal_.
+
+The PowSyBl grid model allows a full representation of the substation connectivity where all the switching devices and busbars are defined (_node/breaker_ level). Automated topology calculation permits to obtain views of the network up to the _bus/branch_ level.
+
+Different states of the network can be stored together with the power system model in an efficient way. The set of attributes that define a given state of the network (steady state hypothesis and state variables) are collectively organized in _variants_. The user can create and remove _variants_ as needed. Setting and getting variant dependent attributes on network objects use the current _variant_.
+
+A set of PowSyBl networks can be merged together in a single network view, and sub-parts of the network model can be easily extracted as separate networks.
```

Comment:
Add or prepare a link (I don't remember if the page about merging already exist). You exclude the destructive merge with this sentence.

Reply:
Indeed the link does not exist. I will prepare the link in that PR asap. @mathbagu: you are right, as destructive merge still exists, please make the difference in a sentence.